src/chief/guide/installation.md

```diff
@@ -2,7 +2,8 @@
 # Installation
 
 ## Package
-Chief can be installed in a blank or existing laravel application. You install the package via composer.
+Chief is a laravel specific package. It can be installed in a fresh or existing laravel application. You install the latest version of the package via composer.
+Laravel version `5.7.1` is a minimum requirement.
```

Comment:
@BenCavens  Hier zeggen we dat 5.7.1 de minimum versie is. Wilt dat zeggen dat de ~5.6.12 weg mag uit de composer file van chief? 
```
"laravel/framework": "~5.6.12|~5.7.1",
```